target/linux/mediatek/filogic/base-files/etc/board.d/02_network

```diff
@@ -165,6 +166,22 @@ mediatek_setup_macs()
 	yuncore,ax835)
 		label_mac=$(mtd_get_mac_binary "Factory" 0x4)
 		;;
+	unielec,u7981-01*)
+		case "$(cmdline_get_var root)" in
+		/dev/mmc*)
+			addr=$(mmc_get_mac_binary factory 0x1000)
+			lan_mac="${addr}"
+			wan_mac=$(macaddr_add "$lan_mac" 1)
+			label_mac=$lan_mac
+			;;
+		*)
+			addr=$(mtd_get_mac_binary factory 0x1000)
```

Comment:
Please use NVMEM in DTS instead for both MAC addresses and label MAC on both, MMC and MTD.